master/buildbot/db/buildrequests.py

```diff
@@ -836,10 +836,11 @@ def thd(conn):
 
         return self.db.pool.do(thd)
 
-    def getBuildChain(self, brid):
```

Comment:
Did you ensure that this method can have always this new parameter? What do you think about making this parameter as additional with default `None` value?

Reply:
There is only 1 calling of this method. With default `None`, I should add next `if` to disable computation `getFriendlyName` if status will be `None`. I think that better way is avoiding unnecessary `'if` here.

![getbuildchain](https://user-images.githubusercontent.com/3229216/39909615-d701df24-54f3-11e8-9388-ed1164c755d2.png)